tests/base/parser/bad/gold/type_t-cid-with.scilla.gold

```diff
@@ -1,12 +1,11 @@
 {
   "errors": [
     {
-      "error_message":
-        "This is an invalid type term, the ADT constructor arguments are likely incorrect. This necessitates a proper separated capital identifier, type identifier, type in brackets or a map.\n",
+      "error_message": "Invalid or incomplete type.\n",
```

Comment:
I think the previous error message (and location) was better, but if you think it's too much of hassle, let's ignore.